src/Roadmap/Roadmap.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { FaLinux } from 'react-icons/fa';
+import { AiFillHtml5 } from 'react-icons/ai';
+import { DiCss3 } from 'react-icons/di';
+import { DiSass } from 'react-icons/di';
+import { FaBootstrap } from 'react-icons/fa';
+import { FaGitAlt } from 'react-icons/fa';
+import { AiFillGithub } from 'react-icons/ai';
+import { FiCodepen } from 'react-icons/fi';
+import { SiNetlify } from 'react-icons/si';
+import { SiJavascript } from 'react-icons/si';
+import { GrReactjs } from 'react-icons/gr';
+import { SiRedux } from 'react-icons/si';
+import { SiTypescript } from 'react-icons/si';
+import { GrNode } from 'react-icons/gr';
+import { DiMongodb } from 'react-icons/di';
+import '../Roadmap/Roadmap.css';
+
+function Roadmap() {
+    return (
+        <div id="body">
+            <h2 className="roadmap-name"><span>Pay IT forward</span> jamoasining
+                 dasturlashni o'rganish xaritasi</h2>
+
+            <div className="solid-box">
+                <div className="solid"></div>
+                <FaLinux className="solid-icon" />
+                <div className="solid"></div>
+            </div>
+
+            <div className="image-box-1">
```

Comment:
Buniyam class nomi boshqalariniki kabi bir xil `image-box` qilishni ilojisini qiling.
Chunki biz bu xaritani JSON datadan quramizda.

Reply:
@Eldorbek7147 mana bunisiga nima deysiz?

Reply:
![image](https://user-images.githubusercontent.com/75212715/119253857-8c823c80-bbcc-11eb-98da-2678940cbb20.png)
 bularni kecha o'zgartirgandimku, yoki siz image-box-1 ni aytyapsizmi, men kecha shu image-box qatnashmaganlariga image-box ni bog'lab to'g'rilab chiqqandim